src/fruity/fruity-host-session.vala

```diff
@@ -393,7 +393,7 @@ namespace Frida {
 		private const uint16 DEFAULT_SERVER_PORT = 27042;
 		private const double MIN_SERVER_CHECK_INTERVAL = 5.0;
 		private const string GADGET_APP_ID = "re.frida.Gadget";
-		private const string DEBUGSERVER_SERVICE_NAME = "com.apple.debugserver";
+		private const string DEBUGSERVER_SERVICE_NAME = "com.apple.debugserver?tls=handshake-only";

```

Comment:
Would be good to move it to where it's used, so this remains just the name of the service.

Reply:
cool! yeah i always underestimate how smart the vala compiler actually is!